hack/docker-builder/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM fedora:27
+FROM fedora:28
 
-ENV LIBVIRT_VERSION 3.7.0
+ENV LIBVIRT_VERSION 4.1.0
```

Comment:
@fabiand here you install a newer libvirt-devel version. We normally let libvirt-devel here and the virt-launcher libvirt version align to avoid issues for virt-launcher when it starts. That in the past always led to issues because virt-launcher could then not use libvirt-client installed in that libvirt version. I see that it does not seem to be the case here, but I don't understand why.

Reply:
virt-launcher still uses libvirt-3.7, also the shared library from libvirt-3.7.